WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -372,6 +372,9 @@ private void endMigration() {
 
     public static void registerMigrationListener(MigrationListener listener) {
         sMigrationListener = listener;
+        if (sIsMigrationError) {
+            sMigrationListener.onError();
```

Comment:
IIUC this should be a controlled case, but can we add a`null` check for `sMigrationListener` here please?